README.md

```diff
@@ -1,20 +1,16 @@
 # cpp_starter_project
 
-[![codecov](https://codecov.io/gh/lefticus/cpp_starter_project/branch/master/graph/badge.svg)](https://codecov.io/gh/lefticus/cpp_starter_project)
-
 [![Build Status](https://travis-ci.org/lefticus/cpp_starter_project.svg?branch=master)](https://travis-ci.org/lefticus/cpp_starter_project)
```

Comment:
This would surely be a broken link if we linked to `cpp-best-practices`.

Reply:
I'll check out codecov.io and appveyor for this move to the organization